markdown/adhoc-backup.md

```diff
@@ -0,0 +1,39 @@
+---
+slug: 'adhoc-backup'
+---
+
+## Adhoc Backup
+
+PMM is able to backup the monitored servers. 
+
+In this section we will show you how can you take Adhoc Backups from a service.
+
+
+### Creating a Backup
```

Comment:
should we mention about limitations we have to run backup?
@artemgavrilov

Reply:
Yes, I was thinking about that but the backup solution is still under development so it will change..  we have to update this doc when the development of the backup process is already done. But let me know what do you think.

Reply:
I don't think that duplicate information is a good idea. We can just provide link to docs where we have requirements: https://docs.percona.com/percona-monitoring-and-management/using/backup.html#mysql-backup-preconditions

Reply:
Cool, can we include that to this doc?